packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# 47.15.0 - January 16th, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Fix source:push commands not appearing when switching to a scratch org without an alias ([PR #1862](https://github.com/forcedotcom/salesforcedx-vscode/pull/1862))
+
+- Prevent output panel from opening automatically for the following commands:
+  - `SFDX: Deploy ...` (all deploy options)
+  - `SFDX: Push ...` (all push options)
+  - `SFDX: Open Default Org`
+  - `SFDX: Authorize an Org`
+  - `SFDX: Authorize a Dev Hub`
+  - `SFDX: Diff File Against Org`
+    ([PR #1865](https://github.com/forcedotcom/salesforcedx-vscode/pull/1865), [Issue #1806](https://github.com/forcedotcom/salesforcedx-vscode/issues/1806))
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Setting to override default max heap size for Apex Language Server ([PR #1817](https://github.com/forcedotcom/salesforcedx-vscode/pull/1817), [Issue #1626](https://github.com/forcedotcom/salesforcedx-vscode/issues/1626))-Contribution by [@kenhuman](https://github.com/kenhuman)
+
+#### salesforcedx-vscode-core
+
+- Support for running `SFDX: Diff File Against Org` on CustomLabels ([PR #1859](https://github.com/forcedotcom/salesforcedx-vscode/pull/1859))
+
```

Comment:
Do we want to add the cache changes for the diff plugin as well? (https://github.com/forcedotcom/sfdx-diff/pull/21)

Reply:
I'm not sure we add release notes here for changes made in plugins/beta plugins.